backend/src/main/java/com/huellapositiva/domain/service/VolunteerProfileService.java

```diff
@@ -79,33 +79,33 @@ public UpdateProfileResult updateProfile(UpdateProfileRequestDto updateProfileRe
     /**
      * Validate profile data.
      *
-     * @param updateProfileRequestDto New user profile information to update
+     * @param profileRequestDto New user profile information to update
      * @param   newEmail True if the user is updating the email
      */
-    private void validations(UpdateProfileRequestDto updateProfileRequestDto, boolean newEmail) {
-        if (newEmail && jpaCredentialRepository.findByEmail(updateProfileRequestDto.getEmail()).isPresent()) {
+    private void validations(UpdateVolunteerProfileRequestDto profileRequestDto, boolean newEmail) {
+        if (newEmail && jpaCredentialRepository.findByEmail(profileRequestDto.getEmail()).isPresent()) {
             throw new EmailAlreadyExistsException("Email already exists in the database.");
         }
-        if (isNotIsland(updateProfileRequestDto.getIsland())) {
+        if (isNotIsland(profileRequestDto.getIsland())) {
             throw new InvalidFieldException("The island field is invalid");
         }
-        if (isNotZipCode(updateProfileRequestDto.getZipCode())) {
+        if (isNotZipCode(profileRequestDto.getZipCode())) {
             throw new InvalidFieldException("The zip code field is invalid");
         }
-        if (isNotPhoneNumber(updateProfileRequestDto.getPhoneNumber())) {
+        if (isNotPhoneNumber(profileRequestDto.getPhoneNumber())) {
             throw new InvalidFieldException("The phone number field is invalid");
         }
-        if (isLengthInvalid(updateProfileRequestDto.getAdditionalInformation())) {
+        if (isLengthInvalid(profileRequestDto.getAdditionalInformation())) {
             throw new InvalidFieldException("The additional information field is invalid");
         }
     }
 
     /**
      * This method update information in location table
-     *  @param updateProfileRequestDto New user credential information to update
+     *  @param profileRequestDto New user credential information to update
      * @param jpaVolunteer JPA representation of the volunteer
      */
-    private void upsertLocation(UpdateProfileRequestDto updateProfileRequestDto, JpaVolunteer jpaVolunteer) {
+    private void upsertLocation(UpdateVolunteerProfileRequestDto profileRequestDto, JpaVolunteer jpaVolunteer) {
```

Comment:
https://github.com/ayudadigital/huelladigital-backend/pull/94#discussion_r606176690

This method can be simplified with the same principle.

Reply:
Done.